gensrc/proto/internal_service.proto

```diff
@@ -267,6 +267,19 @@ message PTabletWriterAddSegmentResult {
     repeated PTabletInfo failed_tablet_vec = 4;
 };
 
+message PRefreshDictionaryCacheRequest {
+    required ChunkPB chunk = 1;
+    required int64 dictionary_id = 2;
+    required int64 txn_id = 3;
+    required POlapTableSchemaParam schema = 4;
+    required int64 memory_limit = 5;
+    required int32 key_size = 6;
```

Comment:
please don't use `required` in thrift and proto

Reply:
done